test/io/reader/YamlReader.test.ts

```diff
@@ -79,6 +79,37 @@ describe('test YamlReader', () => {
     });
   });
 
+  it('should return the parsed specs when they contain wrongly-indented, multiline strings', () => {
```

Comment:
Please add more test cases to cover the same scenario we had with XAP specs.

Reply:
Done ✅